set.mm

```diff
@@ -21197,6 +21197,17 @@ primitive connectives (no defined terms) on the right-hand side.
     dfsb1 ) BCEZBFZBDEZAGZUBAHZBIZHCDEZAGZUFAHZCIZHABDJACDJUAUCUGUEUIUAUBUFATUB
     UFKBBCDLMZNUDUHBCUAUBUFAUJOPQABDSACDSR $.
 
+  ${
+    $d v y $.
+    $( In the case of two successive substitutions for two always equal
+       variables, the second substitution has no effect.  (Contributed by BJ
+       and WL, 9-Aug-2023.) $)
+    aesb2 $p |-
```

Comment:
The ae standing for "all equal" is often added as a suffix, as in cbvaev, hbae for example,  The rest is fine from my POV.

Reply:
Relabeled to sb2ae.